communitiesuk/funding-service-design-assessment-store
# FS 4751 COF25-EOI assessment configs

https://mhclgdigital.atlassian.net/browse/FS-4744
https://mhclgdigital.atlassian.net/browse/FS-4751


### Change description
- Mapping added for COF25-EOI
- Tests added to verify newly added configs
- Inserted cof25-eoi round in to assessment round
- cof25-eoi submitted application seed data added
- Added round ids for HSRA and COF25-EOI




### How to test
- Make sure you have cof25-eoi related roles assigned
- Make and send an application for cof25-eoi
- Login to assessment, validate cof25-eoi configs exist and you can see the received application
- Validate the export options in COF25-EOI work as expected

### Screenshots of UI changes (if applicable)
<img width="1457" alt="Screenshot 2024-10-25 at 09 17 37" src="https://github.com/user-attachments/assets/ab831567-a118-483a-8ade-74055531c7f3">


<img width="1473" alt="Screenshot 2024-10-25 at 09 17 57" src="https://github.com/user-attachments/assets/6f380ffd-688a-4bff-bb97-ec41f55d043f">


<img width="1477" alt="Screenshot 2024-10-25 at 09 18 12" src="https://github.com/user-attachments/assets/2fae6866-4e5e-48b2-9baa-0ab8f96fb844">


<img width="1474" alt="Screenshot 2024-10-25 at 09 18 31" src="https://github.com/user-attachments/assets/e6116fae-f19a-41db-be05-dacc8d4011f9">
